homeworks/roman.vlasov_batteryfree/homework_3/function_array.js

```diff
@@ -0,0 +1,16 @@
+function degree(a) {
+    return a * a;
+}
+function arraystep(a, b) {
```

Comment:
pay attention to naming, JS uses camelCase naming style, in your case I would call function arrayStep